examples/lib/openstreetmap.js

```diff
@@ -0,0 +1,47 @@
+var vumigo = require('vumigo_v02');
+var location = require('../../lib');
+var LocationState = location.LocationState;
+var OpenStreetMap = location.providers.openstreetmap.OpenStreetMap;
+
+var App = vumigo.App;
+var EndState = vumigo.states.EndState;
+
+var LocationApp = App.extend(function(self){
+    App.call(self, 'states:start');
+
+    self.states.add('states:start', function(name){
+        // This state will get the location data from the user and store it in
+        // the contact store. We will store and return the longitude and
+        // latitude information to the user.
```

Comment:
@gsvr No worry. Nice catch. The comment is wrong. My intention was to make the OpenStreetMap example a big simpler than the GoogleMaps one (just to show the simpler use case). I've opened #29 for this.